charts/agent-operator/Chart.yaml

```diff
@@ -2,7 +2,7 @@ apiVersion: v2
 name: grafana-agent-operator
 description: A Helm chart for Grafana Agent Operator
 type: application
-version: 0.1.13
+version: 0.2.0
 appVersion: "0.24.2"
```

Comment:
may as well bump the agent version to `0.25.0`

Reply:
I would suggest to bump the agent version in a different PR, for the following reasons:

- The agent in `0.25.0` starts complaining that CRD `integrations.monitoring.grafana.com` is not found.
- It sounds to be a good moment as well, to sync the CRDs folder from https://github.com/grafana/agent/tree/main/production/operator/crds to the CRDs folder in the chart.
- Since there is a new CRD that the operator handles, upgrade path from `0.24.2` will require some thought? Including the new CRD and performing `helm upgrade --install --force` doesn't install the new CRD, nor update the existing one's.

Reply:
ah yeah that makes sense — since there is a new CRD we should definitely keep it separate